Turnstile-Ontology/99-Utils/Data/Model.yaml

```diff
@@ -11,25 +11,20 @@ ingestion-steps:
 - {nodegroup: "Ingest-HAZARD", csv: "HAZARD.csv"}
 - {nodegroup: "Ingest-HighLevelRequirements", csv: "HighLevelRequirements.csv"}
 - {nodegroup: "Ingest-LowLevelRequirements", csv: "LowLevelRequirements.csv"}
-- {nodegroup: "Ingest-ObjectFile", csv: "ObjectFile.csv"}
 - {nodegroup: "Ingest-Objective", csv: "Objective.csv"}
 - {nodegroup: "Ingest-SoftwareComponentTest", csv: "SoftwareComponentTest.csv"}
 - {nodegroup: "Ingest-SoftwareComponentTestExecution", csv: "SoftwareComponentTestExecution.csv"}
 - {nodegroup: "Ingest-SoftwareComponentTestResult", csv: "SoftwareComponentTestResult.csv"}
 - {nodegroup: "Ingest-SoftwareDesign", csv: "SoftwareDesign.csv"}
 - {nodegroup: "Ingest-SoftwareDesignReview", csv: "SoftwareDesignReview.csv"}
 - {nodegroup: "Ingest-SoftwareDesignReviewArtifacts", csv: "SoftwareDesignReviewArtifacts.csv"}
-- {nodegroup: "Ingest-SoftwareIntegration", csv: "SoftwareIntegration.csv"}
```

Comment:
I'm going to take a look to see what happened with: SoftwareIntegration, SoftwareIntegrationCompileInput, SourceCode, SourceCodeTracing